es6/main.js

```diff
@@ -13,36 +15,31 @@ const shouldAnimate = (anim_queue, timestamp, speed) => {
 
     // // if going forward in time, and future animations exist
     // if (this.time.speed > 0) {
-    //     return (currentAnimations(animations.queue, timestamp, animations.last_timestamp).length
+    //     return (currentAnimations(animations.queue, timestamp, animations.former_time).length
     //             || futureAnimations(animations.queue, timestamp).length)
     // }
     // else if (this.time.speed < 0) {
-    //     return (currentAnimations(animations.queue, timestamp, animations.last_timestamp).length
+    //     return (currentAnimations(animations.queue, timestamp, animations.former_time).length
     //             || pastAnimations(animations.queue, timestamp).length)
     // }
     // return false
 }
 
 
-class AnimationHandler {
-    constructor(store, initial_state, autostart_animating=true) {
-        if (global.requestAnimationFrame) {
-            if (global.DEBUG)
-                console.log('Running animations in a Browser.', {autostart_animating})
+class AnimationsHandler {
+    constructor({store, initial_state, autostart_animating=true, ticker=null}) {
 
-            this.rAF = (func) =>
+        if (ticker === null) {
+            this.ticker = (func) =>
```

Comment:
Can we change the name? `ticker` isn't clear, how about `requestFrameFunc` or `requestTickFunc` or `requestTick`.